spec/parsers/maven_spec.rb

```diff
@@ -93,9 +94,6 @@
         { name: "io.libraries:recursive", requirement: "${recursive.test}", type: "runtime" },
         { name: "io.libraries:optional", requirement: "${optional.test}", type: "runtime", optional: true },
         { name: "io.libraries:not-optional", requirement: "${not-optional.test}", type: "runtime", optional: false },
-        # From dependencyManagement section
-        { name: "org.apache.ant:ant", requirement: "1.9.2", type: "runtime" },
-        { name: "commons-lang:commons-lang",requirement: "2.6", type: "runtime" }
```

Comment:
these were only found in the `<dependencyManagement>` section but not in `<dependencies>`, so Maven wouldn't add these to the project